openapi.yaml

```diff
@@ -6644,6 +6662,7 @@ paths:
             --stackscript_id 10079 \
             --stackscript_data '{"gh_username": "linode"}' \
             --region us-east \
+            --placement_group '{"id": 528, "compliant_only": true}' \
```

Comment:
I think this will need to become `--placement_group.id 528 --placement_group.compliant_only true`

Reply:
I actually missed removing the "compliant_only" functionality from the spec, because it's not supported yet. So, I've set this line to:

'--placement_group.id 528 \` (And, I've changed it everywhere else it occurs.)